src/earn/EarnEnterAmount.tsx

```diff
@@ -386,8 +385,18 @@ function EarnEnterAmount({ route }: Props) {
         inputAmount: tokenAmount.toString(),
         useMax: selectedPercentage === 1,
       })
-    } else {
-      reviewBottomSheetRef.current?.snapToIndex(0)
+    } else if (tokenAmount && prepareTransactionsResult?.type === 'possible') {
```

Comment:
this is just to make TS happy? is the `tokenAmount` check required? seems like there's already a check at the start of the function

Reply:
Oh good call, yeah I think this can be removed